package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "json-ld.org",
+  "private": true,
+  "description": "json-ld.org homepage",
+  "scripts": {
+    "build": "npx @11ty/eleventy",
+    "serve": "npx @11ty/eleventy --serve",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "",
+  "devDependencies": {
+    "@11ty/eleventy": "^2.0.1"
```

Comment:
Any reason to use both `npx` and declare the dependency?

Reply:
No reason.  Switching to remove npx call.